vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -102,29 +102,41 @@ function Get-ContainerImages {
     $imagesToPull = @()
 
     switch ($windowsSKU) {
-        { '2019', '2019-containerd'} {
+        '2019' {
```

Comment:
Is it better to use common images for docker and containerd and images for docker only? 
The images for docker only is the old ones. And when people should add new images to the common image list.

Reply:
This is a workaround. We will combine them later after finding a better solution